src/api/lib/feature_switch/feature.rb

```diff
@@ -0,0 +1,13 @@
+module Feature
+  def self.active?(feature)
+    if User.current && (User.current.is_admin? || User.current.is_staff?)
+      data = Rails.cache.fetch('features', expires_in: 1.hour) do
+        YAML.load_file("#{Rails.root}/config/feature.yml").with_indifferent_access
```

Comment:
Yes! However, currently we have to anyway restart the application to get features enabled as we have auto_refresh option disabled. I thought about implementing it the same way as the gem does but that would be a lot of duplicate code and not sure if this is worth the effort and just caching it for one hour.

> Subsequent calls to Feature will access the cached in-memory representation of the list of features. So changes to toggles in the underlying repository would not be reflected in the application until you restart the application or manually call